### ISSUE TITLE
Ensure command compat with Symfony >= 7.x

### ISSUE THREAD

**Issue by freshworkx** [internetgalerie/ig_slug#40]
**Current Behavior**
I got following error:

```
Fatal error: Declaration of 
Ig\IgSlug\Command\UpdateCommand::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output) must be compatible with Symfony\Component\Console\Command\Command::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output): int in /var/www/html/vendor/internetgalerie/ig-slug/Classes/Command/UpdateCommand.php on line 46
```

**Expected behavior/output**
No error message

**Environment**
TYPO3 version: [12.4.10]
ig_slug version: [1.1.2]
Composer Mode: [yes]

**Possible Solution**
Make execute compatible to new symfony version

**Comment by freshworkx** [comment]
see PR #41

**Comment by freshworkx** [comment]
Merged within v1.1.3 https://github.com/internetgalerie/ig_slug/commit/878c943682552b59123808aa69066ee938150141
Thank you for the quick processing.
